config/core/test-configs.yaml

```diff
@@ -2257,6 +2257,18 @@ test_configs:
   - device_type: meson-gxl-s905x-libretech-cc
     test_plans:
       - baseline
+      - baseline-nfs
```

Comment:
@montjoie Failed in `lab-baylibre`: https://staging.kernelci.org/test/plan/id/62612c99a70a240a9905f4d4/

https://storage.staging.kernelci.org/kernelci/staging-mainline/staging-mainline-20220421.0/arm64/defconfig/gcc-10/lab-baylibre/baseline-nfs-meson-gxl-s905x-libretech-cc.html

However it passed in `lab-broonie`: https://staging.kernelci.org/test/plan/id/6261322de464679a3205f544/